### ISSUE TITLE
crash server on startup

### ISSUE THREAD

**Issue by quentfr21** [webbukkit/DynmapForge#78]
hello

Forge 1.12.2-14.23.3.2681
 Dynmap-2.6-beta-1-forge-1.12.jar

here is the crash log
https://pastebin.com/9Nqhpq8B*

first time after removing the world.pending files the server start again but this time no way

thanks in advance

**Comment by generrosity** [comment]
You can increase the watchdog timeout in your server.Properties ... It might just be that the server is a little under powered and timed out??

**Comment by quentfr21** [comment]
yes i try this and now it's work thanks (timeout probably due to the mod pack)

**Comment by mikeprimm** [comment]
You should try the 1.12.2 specific build - http://dynmap.us/builds/DynmapForge/Dynmap-2.6-alpha-2-forge-1.12.2.jar
